user-guide/rate-limiting-tutorial.md

```diff
@@ -41,7 +41,7 @@ spec:
     name: http-example-rate-limit
     targetPort: http-api
 ---
-apiVersion: apps/v1
+apiVersion: apps/v2
```

Comment:
Sorry should have been clearer. This should remain `v1` as it is a kubernetes api